build_tools/bazel/iree_e2e_generated_runner_test.bzl

```
@@ -106,17 +109,18 @@ def iree_single_backend_e2e_matmul_test(
         target_cpu_features = None,
         timeout = None,
         **kwargs):
-    """Generates an iree_e2e_matmul_test using a custom python generator script.
+    """Generates an iree_e2e_runner_test using a custom python generator script.
 
     The generator script produces .mlir sources which are compiled and passed to
-    iree_e2e_matmul_test.
+    iree_e2e_test.
 
     Args:
         name: Name of the target
+        test_type: Name of the test (e.g., matmuls, conv2ds).
```

match singular/plural used elsewhere
```suggestion
        test_type: Name of the test (e.g., matmul, conv2d).
```

I'd just prefer to use singular everywhere. If we wanted to use this sort of infrastructure for other ops then we could have test types like "attention" or "softmax", and just putting `s` at the end of those is weird ("attentions", "softmaxs")